### PR TITLE
Added support for latest activity sorting

### PR DESCRIPTION
**Pull request by philroche**
Also added better support for github Pull requests

### CODE REVIEW

File: review-gator.py [hunk 1]
```diff
@@ -188,20 +198,28 @@ def get_prs(gr, repo, review_count):
...
             index += 1
             owner = raw_review.user.login
             review = GithubReview(raw_review, raw_review.html_url, owner,
                                raw_review.state, raw_review.submitted_at)
             pr.add_review(review)
+            if pr_latest_activity is None or \
+                            raw_review.submitted_at > pr_latest_activity:
+                pr_latest_activity = raw_review.submitted_at.replace(
+                        tzinfo=pytz.UTC)
+        pr.latest_activity = pr_latest_activity
```
Comment (fginther) [inline review-gator.py]: This code is currently only considering comments associated with votes, which would ignore any comments from the MP submitter. Finding the most recent comment, regardless of vote, should give us a better latest activity metric:

```
        for comment in mp.all_comments:
            if comment.date_created > pr_latest_activity:
                pr_latest_activity = comment.date_created
```

File: review-gator.py [hunk 2]
```diff
@@ -80,29 +81,38 @@ def age(self):
...
+
     def add_review(self, review):
         '''Adds a review, replacing any older review by the same owner.'''
         for r in self.reviews:
             if (review.owner == r['owner'] and review.date > r['date']):
                 self.reviews.remove(r)
                 break
         self.reviews.append(merge_two_dicts(review.__dict__,
-                                            {'age': review.age}))
+                                            {'age': review.age }))
```
Comment (fginther) [inline review-gator.py]: There's an extra space here.

**Comment by philroche** [comment]
Test comment to test age

File: review-gator.py (hunk 2 above)
Comment (philroche) [reply review-gator.py]: Test review comment to test age

**Comment by philroche** [comment]
Test comment age (rev 2)

**Comment by philroche** [comment]
@fginther Now ready for review again with suggested changes

**Pull request merged**